Client/sdk/game/CStreaming.h

```diff
@@ -11,11 +11,24 @@
 
 #pragma once
 
+struct CStreamingInfo
+{
+    WORD  prevId;
+    WORD  nextId;
+    WORD  nextInImg;
+    unsigned char flg;
+    unsigned char archiveId;
+    DWORD offsetInBlocks;
+    DWORD sizeInBlocks;
+    DWORD loadState;
+};
```

Comment:
Is `unsigned char` meant to be a `BYTE`? Not sure which one we should use here. (afaik unsigned char is guaranteed to be exactly a byte. But for the sake of consistency)

Reply:
If this is changed I think we prefer "non-Windows constants"